validphys2/src/validphys/closuretest/closure_results.py

```diff
@@ -394,3 +398,130 @@ def fit_underlying_pdfs_summary(fit, fitunderlyinglaw):
 def summarise_closure_underlying_pdfs(fits_underlying_pdfs_summary):
     """Collects the underlying pdfs for all fits and concatenates them into a single table"""
     return pd.concat(fits_underlying_pdfs_summary, axis=1)
+
+
+def covmat_diffs(data, inconsistent_datasets, sys_rescaling_factor):
+    """Computes the difference between the original covmat and a modified one given:
+        data: complete dataset
+        inconsistent_dataset: list of inconsistent datasets
+
+        return: dictionary containing the relative weight of modification (calculated as diff between traces)
+        """
+
+    dataset_input_list = list(data.dsinputs)
+    commondata_wc = data.load_commondata_instance()
```

Comment:
Here I am not understanding. You are overwriting the content of this variable the line below so what is the reason?